### PR TITLE
MULE-19123: CXF Module fails when request body is empty - Mule 3.9.4

### PR DESCRIPTION
**Pull request by pabloperalta**

### CODE REVIEW

File: modules/xml/src/test/java/org/mule/module/xml/util/XmlUtilsTestCase.java [hunk 1]
```diff
@@ -6,10 +6,16 @@
  */
 package org.mule.module.xml.util;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
```
Comment (szaffarano) [inline modules/xml/src/test/java/org/mule/module/xml/util/XmlUtilsTestCase.java:10]: Avoid using `*`

File: modules/xml/src/test/java/org/mule/module/xml/util/XmlUtilsTestCase.java [hunk 2]
```diff
@@ -6,10 +6,16 @@
...
 import static org.junit.Assert.assertEquals;
+import static org.mule.module.xml.util.XMLUtils.*;
+import static org.mule.util.xmlsecurity.XMLSecureFactories.*;
```
Comment (szaffarano) [inline modules/xml/src/test/java/org/mule/module/xml/util/XmlUtilsTestCase.java:13]: Avoid using `*`

**Review by szaffarano — APPROVED** [review]

File: modules/xml/src/test/java/org/mule/module/xml/util/XmlUtilsTestCase.java (hunk 1 above)
Comment (pabloperalta) [reply modules/xml/src/test/java/org/mule/module/xml/util/XmlUtilsTestCase.java]: Fixed

File: modules/xml/src/test/java/org/mule/module/xml/util/XmlUtilsTestCase.java (hunk 2 above)
Comment (pabloperalta) [reply modules/xml/src/test/java/org/mule/module/xml/util/XmlUtilsTestCase.java]: Fixed

**Pull request merged**